modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -2468,13 +2468,14 @@ CV_EXPORTS_W Mat getAffineTransform( InputArray src, InputArray dst );
 
 /** @brief Retrieves a pixel rectangle from an image with sub-pixel accuracy.
 
-The function getRectSubPix extracts pixels from src:
+The function GetRectSubPix extracts pixels from src:
```

Comment:
Why did you change the function name here?

Reply:
Looking in the documentation
( https://docs.opencv.org/2.4/modules/imgproc/doc/geometric_transformations.html?highlight=getrectsubpix ), we have
`Python: cv2.getRectSubPix(image, patchSize, center[, patch[, patchType]])` and
`Python: cv.GetRectSubPix(src, dst, center) → None`

Since `cv.GetRectSubPix` uses *src, dst and center*, its better to change the function name to GetRectSubPix to avoid any confusion.

Reply:
The latest OpenCV binding for Python is wrapped in the namespace `cv2`. You should
consider `cv2`  first in your new project.

Reply:
Absolutely, But removing information about `dst` could seem like loss in important information in the documentation. 

Also  borderInterpolate() aren't used/mentioned explicitly. It would be better to remove once confirms its non usability.

Reply:
One possible improvement is to change the `dst` in the formula to the output parameter `patch`.